virttest/iscsi.py

```diff
@@ -651,8 +652,20 @@ def export_target(self):
                 self.device,
                 self.emulated_image,
             )
-            output = process.run(device_cmd).stdout_text
-            if "Created %s" % self.iscsi_backend not in output:
+            try:
+                output = process.run(device_cmd).stdout_text
+            except process.CmdError as e:
+                file_exists = re.match(
+                    r".*storage object.*already exists:.*", str(e), re.DOTALL
```

Comment:
During my test with auto case, the error message is "Storage object fileio/device.emulated-iscsi exists". So to cover all related message, maybe we can change it to `file_exists = re.match(
                    r".*storage object.*exists.*", str(e),
                    re.DOTALL | re.IGNORECASE
                    )
`.

Reply:
Thank you, @meinaLi Fixed, please review again.

Reply:
The word "already" and the `:` after exist also need to be removed. Otherwise, the absence of these in the message will cause errors.

Reply:
got it, removed, thanks for re-review